packs/feats/cooperative-soul.json

```diff
@@ -28,7 +28,20 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+              "key": "AdjustDegreeOfSuccess",
+              "selector": "skill-check",
+              "adjustment": {
+                "failure": "to-success",
+                "criticalFailure": "to-success"
+              },
+              "predicate": [
+                "action:aid",
+                "proficiency:expert"
```

Comment:
Probably need to predicate with an or expert, master, legendary. Would be handy if there was a proficiency:rank in the rolloptions so a gte could be used instead.